test/unit/history/toJSON.test.js

```diff
@@ -37,8 +37,11 @@ describe('History::toJSON', function() {
   it('serializes the tree of actions', function() {
     const history = new History({ maxHistory: Infinity })
 
-    history.append(action).resolve(true)
+    let head = history.append(action).resolve(true)
+    let json = JSON.parse(JSON.stringify(history))
 
-    expect(history).toMatchSnapshot()
+    expect(json.head).toEqual(head.id)
+    expect(json.root).toEqual(head.parent.id)
+    expect(json.size).toEqual(2)
```

Comment:
Worth adding a test or two to see that you can access `revisions` and possibly that a revision contains `status`, `payload`, and `timestamp` ?